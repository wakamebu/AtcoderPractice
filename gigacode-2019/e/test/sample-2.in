3 10
1 5
3 5 8
6 1 5
7 2 7
