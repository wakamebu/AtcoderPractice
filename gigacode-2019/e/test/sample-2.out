4.400000000000000355271
